v1.1/stop-a-node.md

```diff
@@ -14,6 +14,11 @@ For information about permanently removing nodes to downsize a cluster or react
 
 ### How It Works
 
+- Finishes in-flight requests.  Note that this is a best effort that times out after the duration specified by the `???` cluster setting (1.1.5)
+- Transfers all *range leases* and Raft leadership to other nodes. (1.1.6)
+- Gossips its draining state to the cluster so that no leases are transferred to the draining node. Note that this is a best effort that times out after the duration specified by the `???` cluster setting, so other nodes may not receive the gossip info in time. (1.1.6)
```

Comment:
This still happens as of 1.1.6. In 1.1.5 and earlier, this part was broken.

Reply:
Thanks for confirming.  Do you know if there is a cluster setting for this in 1.1.6?  None of the [documented cluster settings for 1.1.6](https://www.cockroachlabs.com/docs/stable/cluster-settings.html) look to be the one.  And it wasn't clear from a quick `SHOW ALL CLUSTER SETTINGS` on a 1.1.6 binary (though I may have missed it).